applications/stepswitch/src/stepswitch_inbound.erl

```diff
@@ -18,11 +18,12 @@
 %%--------------------------------------------------------------------
 -spec handle_req(wh_json:object(), wh_proplist()) -> 'ok'.
 handle_req(JObj, _Props) ->
-    'true' = wapi_route:req_v(JObj),
     _ = wh_util:put_callid(JObj),
+    'true' = wapi_route:req_v(JObj),
     case wh_json:get_ne_value(?CCV(<<"Account-ID">>), JObj) of
         'undefined' -> maybe_relay_request(JObj);
-        _AcctID -> 'ok'
+        AccountId -> lager:debug("fetch-id ~s already has account-id ~s, skipping.",
```

Comment:
Additionally, I don't think the app should log if its not handling the request. My personal feeling but others can chime in.